### PR TITLE
Update BTagSF, MET filters, JEC

### PR DESCRIPTION
**Pull request by DouglasBurns**
Latest BTag SF
Latest JEC (v6)
Remove BeamHaloId filter (CSC) to be filtered in txt form in AS
Also to be added in txt form is the 4th Bad EE SC 
WJets sample not TTWJets sample
Prepare for ntuplev5

### CODE REVIEW

**Comment by EmyrClement** [comment]
What is data/BTagSF/._CSVv2.csv ?

**Comment by DouglasBurns** [comment]
It may be something to do with the fact I git deleted the file before
replacing it with the new one. I can remove it.

*________________________*


*Douglas Burns*
*PhD Student*

School of Physics
University of Bristol
HH Wills Physics Lab
University of Bristol
Tyndall Avenue
Bristol
BS8 1TL

 Douglas.Burns@bristol.ac.uk <John.Newby@bristol.ac.uk>

File: src/BristolNTuple_PFJets.cc [hunk 1]
```diff
@@ -30,12 +30,15 @@ BristolNTuple_PFJets::BristolNTuple_PFJets(const edm::ParameterSet& iConfig) :
...
-		reader(		&calib,               // calibration instance
+		reader_b(		&calib,               // calibration instance
 					BTagEntry::OP_MEDIUM,  // operating point
 					"comb",               // measurement type
 					"central"),           // systematics type
-		reader_up(&calib, BTagEntry::OP_MEDIUM, "comb", "up"),  // sys up
-		reader_down(&calib, BTagEntry::OP_MEDIUM, "comb", "down")  // sys down
+		reader_b_up(&calib, BTagEntry::OP_MEDIUM, "comb", "up"),  // sys up
+		reader_b_down(&calib, BTagEntry::OP_MEDIUM, "comb", "down"),  // sys down
+		reader_cl(&calib, BTagEntry::OP_MEDIUM, "mujets", "central"),  // sys down
```
Comment (EmyrClement) [inline src/BristolNTuple_PFJets.cc]: Rename reader_cl to reader_bc.  Same for systematics.

File: src/BristolNTuple_PFJets.cc [hunk 2]
```diff
@@ -30,12 +30,15 @@ BristolNTuple_PFJets::BristolNTuple_PFJets(const edm::ParameterSet& iConfig) :
 
 		// calib("csvv2", "BristolAnalysis/NTupleTools/data/BTagSF/CSVv2.csv"),
 		calib("csvv2", "CSVv2.csv"),
-		reader(		&calib,               // calibration instance
+		reader_b(		&calib,               // calibration instance
```
Comment (EmyrClement) [inline src/BristolNTuple_PFJets.cc]: Rename reader_b to reader_l or reader_light.  Same for systematics.

**Pull request merged**